gnat2goto/driver/tree_walk.adb

```diff
@@ -3771,7 +3774,11 @@ package body Tree_Walk is
          Set_Iter (Body_Loop, Make_Increment (Counter_Sym, Index_Type, 1));
          Set_Lhs (Loop_Test, Counter_Sym);
          Set_Rhs (Loop_Test, Param_Symbol (Len_Arg));
+         Set_Type (I     => Loop_Test,
+                   Value => Make_Bool_Type);
          Set_Cond (Body_Loop, Loop_Test);
+         Set_Init (I     => Body_Loop,
+                   Value => Make_Nil (Sloc (RHS_Element_Type)));
```

Comment:
Have not seen use of aligned name parameters elsewhere - and surprised this passes the compiler? 
If it passes, fine, but there's no precedent for this code style. 
Nitpick.

Reply:
Not sure, but GPS provides this layout as a part of it's auto-completion. I mostly like it, but it can get very verbose sometimes.

Reply:
I've seen this code style in other parts of GNAT.